api/v1alpha1/etcd_types.go

```diff
@@ -69,6 +69,19 @@ type CompressionPolicy string
 // +kubebuilder:validation:Enum=periodic;revision
 type CompactionMode string
 
+type CopySpec struct {
+	// SourceStore defines the specification of the source object store provider for copying backups.
+	// +optional
+	SourceStore *StoreSpec `json:"sourceStore,omitempty"`
+	// MaxAge defines the maximum age in days that one snapshot must be in order to be copied.
+	// By default all snapshots will be copied.
+	// +optional
+	MaxAge *uint32 `json:"maxAge,omitempty"`
+	// SnapshotsToCopy defines how many snapshots will be copied starting with the most recent ones.
+	// +optional
+	SnapshotsToCopy *uint32 `json:"snapshotsToCopy,omitempty"`
```

Comment:
Let's make this consistent with the naming in `etcd-backup-restore`
```suggestion
	MaxSnapshots *uint32 `json:"maxSnapshots,omitempty"`
```